main.go

```diff
@@ -3,14 +3,17 @@ package main
 import (
 	"flag"
 	"fmt"
+	"log"
+	"strings"
+
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/axcdnt/ecs-explorer/services"
-	"log"
-	"strings"
 )
 
 func main() {
+	// aws limits the query in 10 services/request
+	const awsLimitQuerySize = 10
```

Comment:
You could move this up, outside `main` function:
```suggestion
	const maxServicesPerRequest = 10
```

Reply:
Yeah, I think it would be better. I've thought that at first, but then I chose to keep it in function scope. Thanks